### DISCUSSION TITLE
Node Connection Forbidden?

### DISCUSSION THREAD

**Discussion by Silverblade87** [iotaledger/firefly#7700] [Help & Support]
I didnt realize this error before and ive lost my tokens in my firefly. I just created a new wallet through the app on my phone and it immediately gave me this issue, does the app not work? Tried the balance search but its not connecting. Yeah it just keeps sayin node connection forbidden, cant do anything, I originally downloaded the app through IOS but its not letting me do anything and I already sent my tokens to this address and theres nothing there.

txhash:
0xb13f6b02aabc9ced6c7f93d028799e64da84de6ccbd1ed154c5eb6b65a5ba89b

**Comment by begonaalvarezd** [comment]
The mobile app is unmaintained and its currently deprecated. The network was recently upgraded https://blog.iota.org/stardust-upgrade-iota-tokenomics/ so the mobile app doesnt work anymore, as it is not compatible with the new network. You must use the latest desktop version, as of today, version 2.0.3, you can download it from here https://firefly.iota.org/

**Reply by Silverblade87** [reply]
oh great wish I had known that before I tried moving it - so if I just use the desktop version will I still be able to get my tokens thats all I really care about

**Reply by begonaalvarezd** [reply]
Just import your mnemonic or stronghold in the desktop app, if you are missing balances run the balance finder from settings > advanced until all your funds appear

**Reply by Silverblade87** [reply]
got it ill do that thanks
